src/Confluent.Kafka/ConsumeResult.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2017 Confluent Inc., 2015-2016 Andreas Heider
```

Comment:
I'll put it back just in case. The initial comment was in error.

Reply:
related, what do you think about including/excluding `log_level`? it's currently out, and I prefer it that way since people assume it's important but it's not. feel free to convince me otherwise.